src/test/java/com/microsoft/sqlserver/jdbc/AlwaysEncrypted/AESetup.java

```diff
@@ -179,17 +179,19 @@ static void setAEConnectionString(String serverName, String url, String protocol
         if (!isSqlLinux() && null != serverName && null != url && null != protocol) {
             enclaveProperties = "serverName=" + serverName + ";" + Constants.ENCLAVE_ATTESTATIONURL + "=" + url + ";"
                     + Constants.ENCLAVE_ATTESTATIONPROTOCOL + "=" + protocol;
-            AETestConnectionString = connectionString + ";sendTimeAsDateTime=false" + ";columnEncryptionSetting=enabled"
-                    + ";" + enclaveProperties;
+            AETestConnectionString = connectionString + ";sendTimeAsDateTime=false;columnEncryptionSetting=enabled;"
+                    + enclaveProperties;
 
             // show progress if testing multiple servers
             if (enclaveServer.length > 1) {
                 System.out.println("Testing enclave: " + enclaveProperties);
             }
         } else {
-            AETestConnectionString = connectionString + ";sendTimeAsDateTime=false"
-                    + ";columnEncryptionSetting=enabled";
+            AETestConnectionString = connectionString + ";sendTimeAsDateTime=false;columnEncryptionSetting=enabled;";
```

Comment:
Remove as we always have a leading semicolon when we append new properties onto string.

```suggestion
            AETestConnectionString = connectionString + ";sendTimeAsDateTime=false;columnEncryptionSetting=enabled";
```